lib/layouts/user_profile/edit_user_profile.dart

```diff
@@ -244,34 +264,9 @@ class _EditUserProfileState extends State<EditUserProfile> {
               ValueListenableBuilder(
                 valueListenable: placesValue,
                 builder: (context, value, child) {
```

Comment:
since we moved logic to a listener in the initState method why do we need ValueListenableBuilder here?